### DISCUSSION TITLE
Problem with pan.baidu.com

### DISCUSSION THREAD

**Discussion by Loris697** [Charmve/Surface-Defect-Detection#3] [General]
Thanks for sharing this dataset. I've tryed many times to download dataset number 5 and 6 but I cant get them. To download file from Baidu is mandatory to be registered and I cant sign up because italians' telephone number are not supported (it is also mandatory insert a telephone). Could you add the same file on another host site please?

**Comment by Charmve** [comment]
Hi, Loris697

I'm so sorry for this issue. 

You can download this dataset from Google Drive except those can be downloaded from offical website.

link -> https://drive.google.com/drive/folders/1q7lirc_yQBXxUSECwX1UvV1TS4eioFm8?usp=sharing

**Reply by Loris697** [reply]
Thank you so much @Charmve, you saved the day.

**Reply by Charmve** [reply]
Nice day!
